pkg/rancher-desktop/components/MarketplaceCatalog.vue

```diff
@@ -84,7 +84,7 @@ export default (Vue as VueConstructor<Vue & VuexBindings>).extend({
             .includes(this.searchValue.toLowerCase());
         });
       }
-      const filteredExtensions = tempExtensions.filter(item => this.isMobyActive ? item.moby_compatible ?? true : item.containerd_compatible);
+      const filteredExtensions = tempExtensions.filter(item => this.isMobyActive ? true : item.containerd_compatible);
```

Comment:
How is the change to eliminate `item.moby_compatible ?? true` related to removing the Excalidraw extension?

Reply:
Also wouldn't this be a more natural way to write the simplified expression:

```suggestion
      const filteredExtensions = tempExtensions.filter(item => this.isMobyActive || item.containerd_compatible);
```

Reply:
> How is the change to eliminate `item.moby_compatible ?? true` related to removing the Excalidraw extension?

Ok, I got it, this is the only extension using this property, and since it is not TypeScript, there is no reference in a schema or any other reference to the name...

Reply:
Removing this will avoid the tsc error

```
[tsl] ERROR in /home/phillip/Development/rancher-desktop/pkg/rancher-desktop/main/extensions/manager.ts(293,119)
      TS2339: Property 'moby_compatible' does not exist on type '{ id: string; name: string; containerd_compatible: boolean; slug: string; type: string; publisher: { name: string; }; created_at: string; updated_at: string; short_description: string; source: string; ... 8 more ...; filter_type: string; }'.
```

The type for extensions is derived from `_demo_marketplace_items.js`and without Excalidraw in the extensions catalog, `moby_compatible` is no longer a property of `_demo_marketplace_items`.